### PR TITLE
Feature/release2 wa2 b

### PR DESCRIPTION
**Pull request by rayliangatos**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-3615
https://tools.hmcts.net/jira/browse/EUI-3702
https://tools.hmcts.net/jira/browse/EUI-4076
https://tools.hmcts.net/jira/browse/EUI-3940

### Change description ###

This is release candidate for the WA release 2 and 2.1

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by connorpgpmcelroy — DISMISSED** [review]
Approving with very minor comments

**Review by udaydenduluri33 — DISMISSED** [review]
This is the release candidate for the WA release 2 and 2.1

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Should remove mockLocations file - as far as I can tell not being used and it is significantly big

File: api/locations/data/location.mock.data.ts [hunk 1]
```diff
@@ -0,0 +1,13264 @@
+/* tslint:disable:object-literal-sort-keys trailing-comma max-line-length */
+import {LocationModel} from "../models/location.model";
+
```
[the remark this answers is not in the document]
Comment (udaydenduluri33) [reply api/locations/data/location.mock.data.ts]: Yes please. Can you do that?

**Review by connorpgpmcelroy — APPROVED** [review]

**Review by udaydenduluri33 — DISMISSED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]

**Review by sreekanthpuligadda — APPROVED** [review]
LGTM

**Review by udaydenduluri33 — APPROVED** [review]

**Pull request merged**